packages/ember-metal/lib/tags.js

```diff
@@ -19,11 +24,23 @@ export function tagFor(object, _meta) {
   }
 }
 
+function K() {}
+function ensureRunloop() {
+  if (!run) {
+    run = require('ember-metal/run_loop').default;
```

Comment:
@krisselden ^^^See previous note.^^^ (I don't know if you get a notification unless you're tagged.)